cloud/azure/bin/setup-keyvault

```diff
@@ -35,8 +35,12 @@ fi
 echo "Creating resource group ${resource_group}"
 azure::create_resource_group "${resource_group}" "${location}"
 
-echo "Creating key vault ${vault_name}"
-key_vault::create_vault "${resource_group}" "${location}" "${vault_name}"
+if !key_vault::check_if_vault_exists "${resource_group}" "${vault_name}"; then
```

Comment:
i think the spacing here might not be correct?

Reply:
Yes, I think you are correct.  I also just switch the if/else to avoid checking the negative case